src/App.css

```diff
@@ -1,3 +1,325 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
+:root {
+  --Montserrat: 'Montserrat', sans-serif;
+  --RobotoSlab: 'Roboto Slab', serif;
+  --azure: #0290ff;
+  --z: #fafafa;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --cool-blue: #4386bf;
+  --very-light-pink: #c4c4c4;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: var(--white-two);
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 3rem;
+}
+
+.container-title {
+  display: flex;
+  gap: 30px;
+}
+
+nav {
+  display: flex;
+  gap: 2rem;
+}
+
+a {
+  text-decoration: none;
   color: black;
+  opacity: 0.5;
+}
+
+nav a {
+  text-decoration: none;
+  font-family: var(--Montserrat);
+  font-size: 0.813rem;
+  font-weight: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.5;
+}
+
+/* Form styles */
+form {
+  margin: 0 3rem 4rem 3rem;
+  display: flex;
+  gap: 2rem;
+}
+
+input {
+  width: 40%;
+  padding: 1rem 0.5rem;
+  border-radius: 2px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
+  font-family: var(--Montserrat);
+  font-weight: bolder;
+  letter-spacing: -0.15px;
+  color: var(--very-light-pink);
+}
```

Comment:
- [ ] Great effort working on the styling of your Bookstore project, But currently, I would like to highlight some places in your design that require your attention and do not go in line with the original styling.

1. The `book cards`, `input form`, and the `icon` should align together at the right-hand side on a straight line vertically.
2. Also your input form is missing the header test `ADD NEW BOOK` kindly work on that and be sure the text is added.
3. Also the space between one card and the other is much kindly reduce it in order to have a perfect replica of the original design.
4. Lastly, there is a straight line above the `ADD NEW BOOK` text in the original design but that is currently missing in your design.

| Original design | Your design |
| ------------- | ------------- |
| ![book](https://github.com/SaveryIV/Bookstore-React/assets/78634841/c118679d-a0e1-408b-9b0b-cbebe12f89b6) | ![book1](https://github.com/SaveryIV/Bookstore-React/assets/78634841/d6e2f1f6-ddd6-4261-b3fc-09c18c9ca730) |